File: frontend/src/app/data/BlueActionRequest.ts
export class BlueActionRequest {
  constructor(playerUuid: string, cardId: number, inputParams?: number[]) {
    this.playerUuid = playerUuid;
    this.cardId = cardId;
    this.inputParams = inputParams;
  }

  playerUuid: string;
  cardId: number;
  inputParams: number[];

}

File: src/main/java/com/terraforming/ares/model/request/BlueActionRequest.java
package com.terraforming.ares.model.request;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.util.List;
import java.util.Map;

/**
 * Created by oleksii.nikitin
 * Creation date 12.05.2022
 */
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Getter
public class BlueActionRequest {
    private String playerUuid;
    private Integer cardId;
    private Map<Integer, List<Integer>> inputParams;
}
